### PR TITLE
ROX-13263: Add namespaces to deployments search options

### PR DESCRIPTION
**Pull request by janisz**
## Description

We need to add namespaces to search options for deployments to be able to use them in graphql queries and UI.

## Testing Performed

CI

### CODE REVIEW

**Comment by janisz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #3614** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/3614" target="_blank">Graphite</a> 
    * **PR #3615** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/3615" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/3615?utm_source=stack-comment).

**Review by md2119 — CHANGES_REQUESTED** [review]
This PR unnecessarily expands the search option map of deployment. Furthermore, this change does not guarantee correctness. If more search options between deployment and others overlap and deviate, the same issue will be encountered again. 
The correct approach is to ensure `[0]`th options map is of deployment.

**Comment by janisz** [comment]
@md2119 Good catch. Fixed the order and removed `Namespace` from list.

File: ui/apps/platform/cypress/integration/networkGraph/networkGraph.test.js [hunk 1]
```diff
@@ -55,10 +54,7 @@ describe('Network Graph Search', () => {
         });
     });
 
-    it('should filter to show only a specific deployment and deployments connected to it', function () {
-        if (hasFeatureFlag('ROX_POSTGRES_DATASTORE')) {
-            this.skip();
-        }
+    it('should filter to show only a specific deployment and deployments connected to it', () => {
```
Comment (rhybrillou) [inline ui/apps/platform/cypress/integration/networkGraph/networkGraph.test.js:57]: Any reason for the change from `function ()` to `() =>` ?

**Review by rhybrillou — APPROVED** [review]
I'm currently working on a larger-scale re-alignment between pre-postgres and postgres OptionsMaps in GlobalIndex.
That specific fix is part of it. Let's ship it early to open for better postgres test coverage.

**Comment by md2119** [comment]
@janisz could you please re-trigger the CI?

**Review by md2119 — APPROVED** [review]
Some more observations:
- schema.ClustersSchema.OptionsMap is extra for node
- schema.NamespacesSchema.OptionsMap and schema.ClustersSchema.OptionsMap is extra for image

Apart from above, LGTM

File: ui/apps/platform/cypress/integration/networkGraph/networkGraph.test.js (hunk 1 above)
Comment (janisz) [reply ui/apps/platform/cypress/integration/networkGraph/networkGraph.test.js:57]: Linter :)
Apart from that I changed it from `() =>` to `function ()` to be able to access `this` and `skip()` the test.

File: ui/apps/platform/cypress/integration/risk/risk.test.js [hunk 2]
```diff
@@ -161,10 +161,7 @@ describe('Risk page', () => {
             cy.get(RiskPageSelectors.search.searchLabels).should('not.exist');
         });
 
-        it('should have a single URL search param key/value pair in its search bar', function () {
-            if (hasFeatureFlag('ROX_POSTGRES_DATASTORE')) {
-                this.skip();
-            }
+        it('should have a single URL search param key/value pair in its search bar', () => {
```
Comment (pedrottimark) [inline ui/apps/platform/cypress/integration/risk/risk.test.js:164]: You can also remove skip from the next test:
`'should have multiple URL search param key/value pairs in its search bar'`

Comment (janisz) [reply ui/apps/platform/cypress/integration/risk/risk.test.js:164]: Good point! Fixed

**Pull request merged**